coordinator/src/db/rollover_params.rs

```diff
@@ -0,0 +1,98 @@
+use crate::dlc_protocol;
+use crate::schema::rollover_params;
+use bitcoin::Amount;
+use diesel::prelude::*;
+use rust_decimal::prelude::ToPrimitive;
+use rust_decimal::Decimal;
+use time::OffsetDateTime;
+use uuid::Uuid;
+use xxi_node::node::ProtocolId;
+
+#[derive(Queryable, Debug)]
+#[diesel(table_name = rollover_params)]
+struct RolloverParams {
+    #[diesel(column_name = "id")]
+    _id: i32,
```

Comment:
The field needs to be here to satisfy diesel but to prevent clippy from complaining that this field is unused we call it `_id` and map the name to `id`. Kudos to @luckysori because I always added a `clippy::ignore` instead